use-cases/keycloak.md

```diff
@@ -80,30 +86,30 @@ python3 pika-client/producer.py producer kbOFBXI9tANgKUq8vXHLhT6YhbivgXxn
 
 ## Access Management UI
 
-Go to http://localhost:15672, click on the single button on the page which redirects to **Key Cloak** to authenticate.
+Go to http://localhost:15672, click on the single button on the page which redirects to **Keycloak** to authenticate.
 Enter `rabbit_admin` and `rabbit_admin` and you should be redirected back to RabbitMQ Management fully logged in.
 
-
 ## Stop keycloak
 
 `make stop-keycloak`
 
 
 ## Notes about setting up KeyCloak
 
-### Configure JWT signing Keys
+### Keycloak Realm JWKS URI
 
-At the realm level, you go to `Keys > Providers` tab and create one of type `rsa` and you enter the
```

Comment:
Rather than removing this section I would try the following:
1. Start keycloak instance without importing the current configuration but starting from a empty database/configuration
2. Test if the default RSA signing key that comes with keycloak works with RabbitMQ. In keycloak 18.0.0, I had to explicitly add a signing key because the default one did not work. But it has been a while since I did it. 

If the default signing key that comes with keycloak 20.0 works out of box then I agree that we do not need this section anymore. Furthermore, I would use the default `master` realm rather than creating the `test` one as I did it.

Reply:
I would prefer continue using another realm than master, since the master realm is usually used to "manage Keycloak".
I think this resembles more the usual scenarios.

Reply:
I resolved the issue. You can check out the changes done in the last commit in the `main` branch. 
I think there is a better way of solving this issue which is using the "Authorization feature" in Keycloak. It appears as a checkbox that you can turn on/off but in my case, it is disabled and i cannot turn it on. 

This is what I did:
- Create user roles for the read, write and configure scopes
- Assign those roles to the user `rabbit_admin`
- Modify the `realm roles` mapper to map roles to `extra_scope` claim rather than to the existing `scope` claim. 
- Configure rabbitmq to read scopes also from `extra_scope` claim 
- Configure rabbitmq with `jwks_url`. See that I use `keycloak` without no problems there. However, i use `0.0.0.0:8080` in the `oauth_provider_url` because this is what gets delivered to the browser 

Please rebase to take these changes and submit your changes around upgrading to keycloak 20 and import definitions rather than using a binary database.